portal/src/it/scala/cromwell/pipeline/controller/UserControllerItTest.scala

```diff
@@ -23,22 +29,44 @@ class UserControllerItTest
   implicit val config: Config = TestContainersUtils.getConfigForPgContainer(container)
   private val components: ApplicationComponents = new ApplicationComponents()
 
-  override protected def beforeAll(): Unit =
-    components.datastorageModule.pipelineDatabaseEngine.updateSchema()
-
   import components.controllerModule.userController
   import components.datastorageModule.userRepository
 
+  override protected def beforeAll(): Unit = {
+    components.datastorageModule.pipelineDatabaseEngine.updateSchema()
+    val uuid: String = UUID.randomUUID().toString
+    val email: UserEmail = "JohnDoe-@cromwell.com"
```

Comment:
will "John Doe story" be the separate task?

Reply:
Done